write-fonts/src/graph.rs

```diff
@@ -209,6 +211,64 @@ impl Graph {
         }
     }
 
+    /// Write out the serialized graph.
+    ///
+    /// This is not public API, and you are responsible for ensuring that
+    /// the graph is sorted before calling.
```

Comment:
how do I ensure it's sorted? what would happen if I didn't?

Reply:
Not crazy about all the panic pathways, perhaps serialize needs a Result? - write panicking would be Very Bad in some proposed use cases and while I appreciate this _shouldn't_ happen ... we're making it more possible it can.

Reply:
You don't call this method; it's called by something like `dump_table`, which is responsible for creating the graph, trying to pack it, and then serializing if packing is successful.

There isn't actually anything in here that can panic unless the API is misused, and then it should panic every time it gets called.